tests/components/heos/test_init.py

```diff
@@ -92,10 +88,10 @@ async def test_async_setup_entry_loads_platforms(
         assert controller.get_favorites.call_count == 1
         assert controller.get_input_sources.call_count == 1
         controller.disconnect.assert_not_called()
-    assert hass.data[DOMAIN][DATA_CONTROLLER_MANAGER].controller == controller
-    assert hass.data[DOMAIN][MEDIA_PLAYER_DOMAIN] == controller.players
-    assert hass.data[DOMAIN][DATA_SOURCE_MANAGER].favorites == favorites
-    assert hass.data[DOMAIN][DATA_SOURCE_MANAGER].inputs == input_sources
+    assert config_entry.runtime_data.controller_manager.controller == controller
+    assert config_entry.runtime_data.players == controller.players
+    assert config_entry.runtime_data.source_manager.favorites == favorites
```

Comment:
We should not touch internals (`hass.data` or `entry.runtime_data`) in tests. Rather, we should assert if entities are created or if the entry is loaded and so forth.

Reply:
Yeah, removed these lines (and similar from others) as they weren't really necessary for the test itself.